server/src/test/java/io/seata/server/coordinator/DefaultCoordinatorMetricsTest.java

```diff
@@ -63,7 +63,7 @@ public void test() throws IOException, TransactionException, InterruptedExceptio
 
             Assertions.assertEquals(1, measurements.size());
             Assertions.assertEquals(1,
-                    measurements.get("seata.transaction(meter=counter,role=tc,status=active)").getValue(), 0);
+                    measurements.get("seata.transaction(appId=EMPTY_APP,meter=counter,role=tc,status=active)").getValue(), 0);
```

Comment:
same above,if there is no app id ,maybe there is no need to generate this tag (app id tag)